sql/derived_tables/holdings_ext.sql

```diff
@@ -52,11 +27,11 @@ SELECT
     holdings.retention_policy,
     holdings.shelving_title,
     holdings.discovery_suppress,
-    holdings.created_date,
-    holdings.updated_by_user_id,
-    holdings.updated_date
+    json_extract_path_text(holdings.data, 'metadata', 'createdDate') AS created_date,
+    json_extract_path_text(holdings.data, 'metadata', 'updatedByUserId') AS updated_by_user_id,
+    json_extract_path_text(holdings.data, 'metadata', 'updatedDate') AS updated_date
```

Comment:
Cast to date?
```suggestion
    json_extract_path_text(holdings.data, 'metadata', 'updatedDate')::date AS updated_date
```

Reply:
Oddly enough it looks like created_date is a timestamp updated_date is a timestamptz.

In any case, this appears to affect lots of derived tables; so I have made an issue for it:  https://github.com/folio-org/folio-analytics/issues/594

Thanks for catching this.